charts/kong/templates/_helpers.tpl

```diff
@@ -964,7 +964,7 @@ the template that it itself is using form the above sections.
 {{- end -}}
 
 {{- if .Values.admin.ingress.enabled }}
-  {{- $_ := set $autoEnv "KONG_ADMIN_API_URI" (include "kong.ingress.serviceUrl" .Values.admin.ingress) -}}
+  {{- $_ := set $autoEnv "KONG_ADMIN_GUI_API_URL" (include "kong.ingress.serviceUrl" .Values.admin.ingress) -}}
```

Comment:
There's no harm in still setting the old variable, correct? The chart is intended for multiple Kong versions, so we should set both unless the old value breaks things. A brief test indicates you can set both; it just warns about the deprecated value on 3.4+

```suggestion
  {{- $_ := set $autoEnv "KONG_ADMIN_GUI_API_URL" (include "kong.ingress.serviceUrl" .Values.admin.ingress) -}}
  {{- $_ := set $autoEnv "KONG_ADMIN_API_URI" (include "kong.ingress.serviceUrl" .Values.admin.ingress) -}}
```

Reply:
yes that makes sense. I am using current version (2.26.3) of chart with both variables set and its running fine. We can keep both.

Reply:
yes that makes sense. I am using current version (2.26.3) of chart with both variables set and its running fine. We can keep both.